engine/tests/core38/entity/test_entity.py

```
@@ -0,0 +1,200 @@
+            return []
+
+        ctx = SimpleNamespace(
+            get_registries=get_registries,
+            get_inline_behaviors=lambda: [],
+        )
+        with using_ctx(ctx):
+            _ = Entity(label="original")
+        assert calls["registries"] == 1
+
+    def test_structure_without_ctx_no_dispatch(self) -> None:
+        on_create(func=lambda *, data, **_: {"label": "mutated"})
```

Ruff (0.15.0)

[warning] Unused lambda argument: `data`

(ARG005)